<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentWidget</class>
 <widget class="QWidget" name="CurrentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>输出电流</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>    （输出电流设定及标定）</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>仪表电流</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>仪表当前电流（mA）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="currentlineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>输出电流百分比</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="currentPercentlineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="queryCurrentButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>固定电流输出</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="1">
          <widget class="QLineEdit" name="currentSetlineEdit"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="currentChoiceButton">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="currentSetButton">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>其他电流</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="currentChoicecomboBox">
           <item>
            <property name="text">
             <string>4mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16mA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20mA</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>电流选择</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="currentQuiteButton">
           <property name="text">
            <string>退出固定电流模式</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="title">
      <string>电流输出校准</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="2">
       <widget class="QLabel" name="label_34">
        <property name="text">
         <string>mA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>注意：要先设置固定电流4mA输出，设置值应在2mA与6mA之间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="currentLowSetlineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="currentLowSetButton">
        <property name="text">
         <string>调整</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="currentUpSetlineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>输出低电流微调：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>mA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>注意：要先设置固定电流20mA输出，设置值应在18mA与22mA之间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>输出高电流微调：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="currentUpSetButton">
        <property name="text">
         <string>调整</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="title">
      <string>校正说明</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;电流输出校正时必须先校正4mA，然后再校正20mA，注意调整按钮后的说明。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;校正方法：输出低电流校正时首先通过上面的固定电流输出设置仪表固定输出4mA，然后测量电流值，测得的电流值即为所要输入的校正值。例如设置固定输出4mA后仪表显示输出电流为4mA，但是实际测量值为5mA，则在微调输入框中输入5然后点击调整按钮即可校正。高电流校正过程同理。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
